recipe/meta.yaml

```
@@ -77,53 +80,77 @@ outputs:
         # so that one could run cmake for generating *.bc, and other
         # common data files.
         - arrow-cpp ={{ arrow_version }}=*cpu
-        - bisonpp
+        - bisonpp  # [linux64]
         - blosc
         - boost-cpp
         - clangdev {{ llvm_version }}
         - double-conversion
-        - flex
+        - flex  # [linux64]
         - fmt
         - geos
         - glog
         - llvmdev {{ llvm_version }}
         - llvm {{ llvm_version }}
         - libarchive
+        - libevent  # [win]
         - libgdal
         - libpng
         - librdkafka
+        - tbb
+        - tbb-devel
         - thrift-cpp {{ thrift_version }}
+        - winflexbison  # [win]
     test:
       commands:
         # Test installation
         # doc
-        - test -f ${PREFIX}/share/doc/omnisci/LICENSE.md
+        - test -f ${PREFIX}/share/doc/omnisci/LICENSE.md  # [linux64]
```

```suggestion
        - test -f ${PREFIX}/share/doc/omnisci/LICENSE.md  # [not win]
```
The same applies to all changes below in order to support osx target as well.